package.json

```diff
@@ -44,28 +44,31 @@
     "cleanup:hard": "rm -rf ./dist && rm -rf ./node_modules && rm -rf ./package-lock.json"
   },
   "dependencies": {
-    "vue": "^2.7.10"
+    "vue": "^3.4.15"
   },
   "peerDependencies": {
     "axios": "^1.3.3",
     "lodash": "^4.17.21",
     "luxon": "^3.2.1",
+    "moment": "^2.30.1",
```

Comment:
where is moment used?

Reply:
at articles-accordion.vue component so for

Reply:
lets use luxon, as its already referenced in the lib, https://github.com/scbd/scbd-common/blob/master/src/services/filters/datetime.js